drivers/periph_common/init.c

```diff
@@ -19,6 +19,7 @@
  */
 
 #include "periph/spi.h"
+#include "periph/rtc.h"
```

Comment:
this should be guarded with `#if RTC_NUMOF` as well.

and in theory the same for the spi header above, too - see #7079